src/app/results/results.component.ts

```diff
@@ -188,5 +188,6 @@ export class ResultsComponent implements OnInit {
 
   ngOnInit() {
     document.getElementById('nav-group').style.width = '632px';
+    document.getElementById('nav-input').style.width = '536px';
```

Comment:
Okay, @nikhilrayaprolu working on it. :+1: The typescript code will look more good.